CHANGELOG.md

```diff
@@ -3,7 +3,10 @@
 0.3 (unreleased)
 ----------------
 
-- Nothing changed yet.
+- The JSON is saved in the same folder as the active manholes layer
+- Users are asked for their username and password (username is entered by default) when uploading voortgang or final version.
```

Comment:
Zelf doe ik altijd lege regels tussen de bullets.
En ook de changelog op 78 karakters uitlijnen.

En als je het arme ding ook nog van `.md` in `.rst` kunt hernoemen wordt ik helemaal blij :-)